.cargo/config

```diff
@@ -1,2 +1,2 @@
-[build]
-rustflags = ["-C", "relocation-model=pic", "--emit=obj"]
\ No newline at end of file
+[target.aarch64-unknown-linux-gnu]
+linker = "aarch64-linux-gnu-gcc"
```

Comment:
had to specify the arm64 linker for arm builds. Also no longer needed that PIC relocation model config.